README.md

```diff
@@ -2,3 +2,7 @@
 
 This repo contains a JSON marshaler suitable for use with the gRPC-Gateway
 when using `gogo/protobuf` types.
+
+## License
+
+95% of the code is copied from the gRPC-Gateway project, so their license applies.
```

Comment:
Is this true?
Just checking.

Reply:
> Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

Easiest just to copy the LICENSE.

Reply:
https://github.com/grpc-ecosystem/grpc-gateway/blob/master/LICENSE.txt